### ISSUE TITLE
An error in eval.py

### ISSUE THREAD

**Issue by YinZhe0412** [mahmoodlab/CLAM#174]
Dear,
The eval.py has not been modified，but the file csv is changed to Step_3.csv.
[Step_3.csv](https://github.com/mahmoodlab/CLAM/files/10729424/Step_3.csv)
#166 

The error is that the later independent test set is used for testing, but the pt file of the training set appears (can be found in Step_3. csv). This is something I can't understand. Sorry, I am a beginner in this field.

The error:
![image](https://user-images.githubusercontent.com/81342165/218656137-606e7ccf-94b5-4a00-8a7e-43a91dffc405.png)

splits_0.csv:
![image](https://user-images.githubusercontent.com/81342165/218655950-82be64e8-ae40-4824-aa21-b7b8857fc0a9.png)
Thank you!

**Comment by fedshyvana** [comment]
sorry if the code seems unintuitive, but if you change the csv file, you need to modify the eval.py file so that a dataset using the updated csv file is defined.

**Comment by YinZhe0412** [comment]
How should we modify eval.py? Which parts need to be modified? Thank you very much for answering me.

**Comment by YinZhe0412** [comment]
![image](https://user-images.githubusercontent.com/81342165/219910492-fe84913b-ce1e-4f42-b60b-3e380b853048.png)
Sorry, I really don't understand this.
